feature: add next.js, react quickstarts (#1093)
knocklabs/docs: content/getting-started/quick-start/react.mdx

---
title: Get started with Knock
description: Quickly get up and running with Knock and React.
tags: ["getting started"]
section: Getting started
---

> In this guide, you'll integrate Knock's `NotificationFeed` component a React application and send your first notification using Knock's test runner.

Reviewer: Like what Scoti called out above, we'll want to remove our use of "guide" here
```suggestion
This page covers how to integrate Knock's `NotificationFeed` component a React application and send your first notification using Knock's test runner.
```
